components/match2/wikis/rocketleague/match_summary.lua

```diff
@@ -41,15 +42,19 @@ local Casters = Class.new(
 
 function Casters:addCaster(caster)
 	if Logic.isNotEmpty(caster) then
-		table.insert(self.casters, '[[' .. caster .. ']]')
+		if caster['flag'] then
+			table.insert(self.casters, Flags.Icon(caster['flag']) .. ' [[' .. caster['name'] .. ']]')
+		else
+			table.insert(self.casters, '[[' .. caster['name'] .. ']]')
```

Comment:
adjust display to use the id of the caster instead of the link

```suggestion
		local nameDisplay = caster.id or caster.name
		nameDisplay = '[[' .. caster.name .. '|' .. nameDisplay .. ']]'
		if caster.flag then
			table.insert(self.casters, Flags.Icon(caster['flag']) .. ' ' .. nameDisplay)
		else
			table.insert(self.casters, nameDisplay)
```